idd/Energy+.idd.in

```diff
@@ -52999,9 +53001,13 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key Load
        \key SetPoint
+       \key ASHRAE90VariableFan
```

Comment:
Correct. Both SingleZoneVAV and Load require a control zone name input. I'll make the change.